acceptable/_service.py

```diff
@@ -113,9 +119,6 @@ def __init__(self, name, group=None, metadata=None):
         :raises TypeError: If the name string is something other than a
             string.
         """
-        if not isinstance(name, str):
-            raise TypeError(
-                "name must be a string, not %s" % type(name).__name__)
```

Comment:
I think this method's docstring (the `:raises TypeError:` part) is now wrong.

(Alternatively, retain this check but use `past.builtins.basestring`?)